pkg/utils/utils.go

```diff
@@ -179,38 +180,15 @@ func RemoveDuplicates(elements []string) []string {
 	return result
 }
 
-const termjsBin string = "termjs-cli"
-
-// AttachToSerial tries to connect to server serial using 'term.js-cli' and fallback with a help message
+// AttachToSerial tries to connect to server serial using 'gotty-client' and fallback with a help message
 func AttachToSerial(serverID string, apiToken string, attachStdin bool) error {
-	termjsURL := fmt.Sprintf("https://tty.cloud.online.net?server_id=%s&type=serial&auth_token=%s", serverID, apiToken)
+	URL := fmt.Sprintf("https://tty.cloud.online.net/?arg=%s&arg=%s", apiToken, serverID)
 
-	args := []string{}
-	if !attachStdin {
-		args = append(args, "--no-stdin")
-	}
-	args = append(args, termjsURL)
-	log.Debugf("Executing: %s %v", termjsBin, args)
-	// FIXME: check if termjs-cli is installed
-	spawn := exec.Command(termjsBin, args...)
-	spawn.Stdout = os.Stdout
-	spawn.Stdin = os.Stdin
-	spawn.Stderr = os.Stderr
-	err := spawn.Run()
+	gottycli, err := gottyclient.NewClient(URL)
```

Comment:
We will take advantage of this migration to pass context streams:

- [ ] blocked by https://github.com/moul/gotty-client/issues/21
- [ ] update AttachToSerial to take `ctx` or streams as argument